### ISSUE TITLE
add to description, how to connect  J1 to Junkers 1-2-4 interface

### ISSUE THREAD

**Issue by renarena** [Strooom/SmartTherm#6]
Please, can you add the junkers 1-2-4 contacts in https://github.com/Strooom/SmartTherm/blob/master/hardware/SmartTherm-v2-schematics.pdf or add a note in the docs 
to show J1 contacts 1 2 3 4 does not correspond to Junkers 1-2-4?

J1-1 = GND = Junkers 4
J1-4 = +24V = Junkers 1

**Comment by Strooom** [comment]
You are right - it would have been better if the numbering on the SmartTherm connector was matching the numbering on the Junkers boilers. I will correct that in the next version.

In the mean time, here are the correct pin numberings on both sides:
Junkers 1 = 24V = SmartTherm 4
Junkers 2 = drive signal = SmartTherm 2 & 3
Junkers 4 = GND = SmartTherm 1
